XCUITests/FxScreenGraph.swift

```diff
@@ -216,7 +258,7 @@ func createScreenGraph(_ app: XCUIApplication, url: String = "https://www.mozill
         scene.gesture(to: PasscodeSettings) {
             if app.navigationBars["Require Passcode"].exists {
                 // Go back, accepting modifications
-                app.navigationBars["Require Passcode"].buttons["Passcode"].tap()
+                app.navigationBars["Require Passcode"].buttons["Passcode For Logins"].tap()
```

Comment:
Please add `accessibilityIdentifier` to any views that need them.

Reply:
Hmm, just realized "Require Passcode" and "Enter Passcode" are identifiers.